pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -73,14 +76,21 @@ stdenv.mkDerivation rec {
     # Add extra configuration
     mkdir $out/etc/bluetooth
     ln -s /etc/bluetooth/main.conf $out/etc/bluetooth/main.conf
+
+    # Add missing tools, ref https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/bluez
```

Comment:
This doesn't seem like something we should be doing- but I guess if Arch thinks its a good idea. Could you try doing this instead though:

```
substituteInPlace Makefile.tools --replace noinst_PROGRAMS bin_PROGRAMS
```

Not sure if it will work but IMO much cleaner than the for loop.

Reply:
> This doesn't seem like something we should be doing

Why not some of these tools might be experimental but I fail to see why making them available is bad in any way. It's not a change of behaviour.

The ```substituteInPlace``` didn't do the trick unfortunately (it would indeed have been much cleaner).

Reply:
I guess the main question is: why didn't the maintainers have these installed by default? It could just be  a bug but there might also be a valid reason. Definitely not a blocker for this - I am just skeptical of custom install scripts in general.

Reply:
Based on my reading of the ML archives, the bluez guys deprecated the existing tools before having the replacements ready and while these tools are experimental, they are needed for certain tasks (example: toggle an adapter to LE mode only) that were previously the responsibility of ```hciconfig``` and ```hcitool```.

The existence of them doesn't mean that the bluetooth daemon starts doing stuff differently so everything should work as normal but we now make low-level manipulation of the bluetooth stack/adapters possible again.